baselines/disable-packing-test/src/v1beta1/echo_client.ts.baseline

```diff
@@ -24,7 +24,7 @@ import { Transform } from 'stream';
 import { RequestType } from 'google-gax/build/src/apitypes';
 import * as protos from '../../protos/protos';
 import * as gapicConfig from './echo_client_config.json';
-
+import * as operationProtos from 'google-gax/build/protos/operations';
```

Comment:
We should make sure whatever we need is exported from index in gax.  This pattern gets you into breaking-change trouble far too easily.

Reply:
yes. this can be easily exported from gax, thanks